.github/workflows/production.yml

```diff
@@ -21,8 +21,8 @@ env:
   K8S_LIMIT_CPU: 400m
   K8S_LIMIT_RAM: 400Mi
   K8S_REPLICACOUNT: 2
-  K8S_LIVENESS_PATH: /api/healthz
-  K8S_READINESS_PATH: /api/readiness
+  K8S_LIVENESS_PATH: /client/healthz
```

Comment:
This is probably wrong?

Reply:
Yup, seems that my IDE derped a ton with this rename. First the functianility would't go through and now it seems it fixed automatically all the wrong files and didn't touch the ones that actually needed fixing.